src/shared/models/UpdatePlayerOptions.ts

```diff
@@ -11,7 +11,7 @@ export interface UpdatePlayerOptions {
     sdk                         ?: string;
     device_model                ?: string;
     app_id                      ?: string;
-    tags                        ?: Object | null;
+    tags                        ?: Record<string, unknown> | null;
```

Comment:
what about this record, we know this should be a string, correct?

Reply:
ah yes, good catch! Fixed now